airflow/models/taskinstance.py

```diff
@@ -501,6 +524,13 @@ def refresh_from_task(self, task, pool_override=None):
         self.max_tries = task.retries
         self.executor_config = task.executor_config
         self.operator = task.__class__.__name__
+        # Mocks tags attribute is not iterable
+        if isinstance(task.task_tags, coll.Iterable):
+            for tag in task.task_tags:
+                self.task_tags.append(
+                    TaskTag(name=tag, dag_id=self.dag_id,
+                            task_id=self.task_id, execution_date=self.execution_date)
+                )
```

Comment:
Doesn't SQLA handle this automatically for the relationship?
```suggestion
                self.task_tags.append(TaskTag(name=tag))
```

Reply:
Apparently it does. Fixed in 9e683c4